driver/bindings/junit/src/main/scala/org/virtuslab/ideprobe/IntelliJFixture.scala

```diff
@@ -48,6 +48,8 @@ final case class IntelliJFixture(
 
   def deleteWorkspace(workspace: Path): Unit = {
     workspace.delete()
+    val workspaceBase = workspace.getParent
+    if (Files.isDirectory(workspaceBase) && workspaceBase.toFile.list().isEmpty) Files.delete(workspaceBase)
```

Comment:
or you could just do: `workspace.getParent.delete()` instead of `workspace.delete()`. It should still be a directory controlled by the probe, so it should be safe to delete

Reply:
My intent was to make it clear that the parent directory is assumed to be empty. If the directory is not empty (which seems to happen in some tests), that's an unexpected situation, and I'd rather leave it as it is than risk deleting important data.

Reply:
I refactored workspace setup a bit. Now WorkspaceTemplate contains the cleanup logic. I believe Marek is right. If we setup directory in directory as a workspace, we should also clean it up, nothing strange in this logic. 
In fact for fastpass tests, the `ws.parent` will not be empty, as fastpass will setup <ws>/bsp-projects along with <ws>/ws, so it is desirable to have it deleted together.